distribution/lib/Standard/Base/0.0.0-dev/src/System.enso

```diff
@@ -1,9 +1,13 @@
-## Functionality for interacting with the host system.
-
-from Standard.Base import all
+import project.Data.Array.Array
+import project.Data.Boolean.Boolean
+import project.Data.Numbers.Integer
+import project.Data.Text.Text
+import project.Nothing.Nothing
 
 polyglot java import java.lang.System as Java_System
 
+## Functionality for interacting with the host system.
```

Comment:
What does this comment refer to?

If it's a comment for the whole file, I think it makes sense to keep it as it was originally - at the very top. It's much harder to spot it below the imports.